packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+
+export const alertType = ['success', 'danger', 'warning', 'info'];
+
+const propTypes = {
+  variant: PropTypes.oneOf(alertType).isRequired,
+  action: PropTypes.node,
+  ariaLabel: PropTypes.string,
+  title: PropTypes.string,
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  ariaLabel: '',
+  action: null,
+  title: '',
+  children: '',
+  className: ''
+};
+
+const Alert = ({
+  variant,
+  ariaLabel,
+  action,
+  title,
+  children,
+  className,
+  ...props
+}) => (
+  <div
+    {...props}
+    className={css(
+      styles.alert,
+      getModifier(styles, variant, styles.modifiers.info),
+      className
+    )}
+    aria-label={ariaLabel}
```

Comment:
We should default this to `${variant} Notification` that way by default users get an accessible experience.